front/pages/w/[wId]/assistant/labs/transcripts/index.tsx

```diff
@@ -196,6 +207,8 @@ export default function LabsTranscriptsIndex({
           ...prev,
           provider: transcriptsConfiguration.provider || "",
           isGongConnected: transcriptsConfiguration.provider == "gong" || false,
+          isModjoConnected:
```

Comment:
So you can only connect one unique meeting transcript provider?

Reply:
Per person, yes